skills/nodej/bulk-update-skills.js

```diff
@@ -0,0 +1,49 @@
+const execSync = require('child_process').execSync;
+
+let removeSkills = ['Juggling','Fire Breathing','Origami'];
+
+console.log('Working on it...');
+
+// Loop through each skill to be deleted
+removeSkills.forEach(skill => {
+    let gcBuilder = 'gc';
+    let jqBuilder = 'jq -c -r \'[.[] | select(.skills[].name=="'+ skill +'")]\''
+
+    if(process.platform === 'win32') {
+        gcBuilder = 'gc.exe';
+        jqBuilder = 'jq -c -r "[.[] | select(.skills[].name==""'+ skill +'""")]"';
```

Comment:
From the [JQ Downloads page](https://stedolan.github.io/jq/download/) it looks like JQ on Windows is named either `jq-win64.exe` or `jq-win32.exe` depending on the architecture.
If you don't have access to a Windows machine it might be a good idea to ask Prince for a review of the Windows functionality. I'm pretty sure he has one because he added some PowerShell examples.

If it's too awkward to get this working for Unix and Windows we can always just leave it working for Unix and have it exit if running on Windows.

Reply:
I agree with Roman, if we cant do it, I would have Prince do an example in powershell

Reply:
Hi John, Prince and I have put together a way to also have this code run in Windows. Changed codes can be found in the latest commit. :)

Reply:
I was thinking about this and maybe it would be better to try find what the JQ executable is called on windows.
You could try calling execSync on jq-win64 and if the error code indicates it doesn't exist go to jq-win32 and then to jq. If none are found then error out with a message telling them to download it.

Reply:
Ronan.  I think this is good enough.  Any decent developer will figure it.  :)